scripts/exregional_run_analysis.sh

```diff
@@ -830,6 +858,14 @@ cat << EOF > gsiparm.anl
 $gsi_namelist
 EOF
 
+if [[ ${gsi_type} == "ANALYSIS" && ${ob_type} != "AERO" ]]; then
+# base_e GSI can't handle the below two parameter for RRFS-SD DA. So delete them.
+  mv gsiparm.anl gsiparm.anl.sd
+  sed '/laeroana_fv3smoke/d' gsiparm.anl.sd > gsiparm.anl.1
+  sed '/berror_fv3_cmaq_regional/d' gsiparm.anl.1 > gsiparm.anl
+  rm -fr gsiparm.anl.sd gsiparm.anl.1
+fi
+
```

Comment:
This requires GSL GSI is rebased to the latest EMC GSI development branch. If not, we have to delete the parameters related to SMOKE DA to make the meteorological DA run. That is one reason I added a separated *sh to produce gsiparm.anl for SD DA in my previous PR.

Reply:
Is it possible to update GSL GSI with those two namelist parameters according to the EMC/develop branch? Don't need to add the full functionality at the moment, but only to add placeholders so that GSL GSI be able to proceed with the new namelist.

Reply:
@guoqing-noaa 
I think it would be good I keep the lines in my personal application or add the two parameters in GSL/GSI. I will delete the lines in this PR.

Reply:
FYI, I just checked the latest GSL/GSI (the feature/rrfs_dev branch). It already included the following two parameters:
```
berror_fv3_cmaq_regional
laeroana_fv3smoke
```

Reply:
@guoqing-noaa 
That is great. My current test is based base_e, which doesn't accept the two namelist parameters. I may use the lastest GSL GSI rrfs_dev in my next test.

Reply:
Sorry that although berror_fv3_cmaq_regional is already there and we also have the laeroana_fv3smoke variable in chemmod.f90, laeroana_fv3smoke is not added in gsimod.f90 yet. I believe it was added only in your own GSI copy at the moment. Could you create a PR to update GSL/GSI with this? (otherwise it will stop the normal RRFS workflow). Thanks!